app/views/content/terms_candidate.md

```diff
@@ -10,69 +10,68 @@ You must not:
 - gather or use data from our website
 - use abusive language or act abusively towards anyone you come into contact with
 
-## Applying for teacher training courses
+## Applying for courses
 
-### Stage 1: Submitting your initial application
+### Submitting your initial application
 
-You can apply for a total of 3 courses.
+You can apply for up to 3 courses.
 
-### Stage 2: Apply again if your application was not successful
+### Apply again if your application does not lead to a place
 
-You can apply for another course at this stage if you do not get an offer or if you choose not to accept your offers.
+You can apply for another course if your initial application does not lead to a place.
 
-[<%= t('service_name.get_into_teaching') %>](<%= t('get_into_teaching.url') %>) can guide you through the process of applying for more courses. Contact them for free on <%= t('get_into_teaching.tel') %>, or [chat to an adviser using the online chat service](<%= t('get_into_teaching.url_online_chat') %>).
+This might be because you:
 
-### Making changes to your application
-
-Once you submit your application it is automatically sent to the training provider. To change your application after you’ve submitted it, contact us as soon as possible at <%= bat_contact_mail_to %>.
+* do not get an offer
+* decline any offers
+* withdraw your application
 
-You can ask us to add courses (if you have not reached your limit of 3) and change your name and contact details too.
+You can apply again to one course at a time as many times as you like.
 
-If you have heard back from your training provider, for example if they have turned down your application, we will not be able to change your choice of course.
-
-### Withdrawing your application
+[Chat online for help applying again](<%= t('get_into_teaching.url_online_chat') %>).
 
-You can withdraw your application at any point, even after you have accepted an offer.
+### Making changes to your application
 
-[Sign in to your account](<%= candidate_interface_sign_in_path %>) and click ‘withdraw’ next to the course you want to withdraw.
+Once you submit your application it is sent to your teacher training providers straight away.
 
-### Outcome of your application
+Contact us within 5 working days at <%= bat_contact_mail_to %> if you need to change something.
```

Comment:
Ah, so we're tackling a common issue here on the 5 day rule. Perhaps we can bundle in changes to the terms of use alongside this?